Modules/_ctypes/_ctypes.c

```diff
@@ -3262,7 +3262,8 @@ PyCFuncPtr_FromDll(PyTypeObject *type, PyObject *args, PyObject *kwds)
         /* Here ftuple is a borrowed reference */
         return NULL;
 
-    if (!PyArg_ParseTuple(ftuple, "O&O", _get_name, &name, &dll)) {
+    if (!PyArg_ParseTuple(ftuple, "O&O;illegal func_spec argument",
+                          _get_name, &name, &dll)) {
```

Comment:
In case of multiline condition the opening `{` should be moved to the separate line for better readability. This is a new requirement of PEP 7.

This comment is applicable to other changes below.